code/modules/mob/living/simple_animal/hostile/syndicate.dm

```diff
@@ -238,7 +238,7 @@
 	response_help = "pokes"
 	response_disarm = "shoves"
 	response_harm = "hits"
-	speed = 6
+	speed = 2
 	move_to_delay = 1
 	turns_per_move = 1
```

Comment:
```suggestion
```

Reply:
Oh and reason is that its defined for 5 just at line 237 iirc

Reply:
That makes them too slow.

Reply:
then maybe remove the turns_per_move = 5 var at 237?